src/NerdMinerV2.ino.cpp

```diff
@@ -31,6 +39,20 @@ unsigned long start = millis();
 
 //void runMonitor(void *name);
 
+void alternate_screen_state() {
+  int screen_state= digitalRead(38);
+  //Serial.printf("Screen state is '%s', switching to '%s'", screen_state, !screen_state);
+  Serial.println("Switching display state");
+  digitalWrite(38, !screen_state);
```

Comment:
there is a macro defined for the back light pin : TFT_BL
```suggestion
  int screen_state= digitalRead(TFT_BL);
  //Serial.printf("Screen state is '%s', switching to '%s'", screen_state, !screen_state);
  Serial.println("Switching display state");
  digitalWrite(TFT_BL, !screen_state);
```

Reply:
Done